test/uk/gov/hmrc/incometaxproperty/controllers/PeriodicSubmissionControllerSpec.scala

```diff
@@ -0,0 +1,84 @@
+/*
+ * Copyright 2023 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.incometaxproperty.controllers
+
+import play.api.http.Status.{INTERNAL_SERVER_ERROR, NOT_FOUND, OK}
+import play.api.libs.json.Json
+import play.api.test.Helpers.status
+import uk.gov.hmrc.incometaxproperty.models.errors.{ApiServiceError, DataNotFoundError}
+import uk.gov.hmrc.incometaxproperty.models.{PeriodicSubmission, PeriodicSubmissionResponse}
+import uk.gov.hmrc.incometaxproperty.utils.ControllerUnitTest
+import uk.gov.hmrc.incometaxproperty.utils.mocks.{MockAuthorisedAction, MockPropertyService}
+import uk.gov.hmrc.incometaxproperty.utils.providers.FakeRequestProvider
+
+import scala.concurrent.ExecutionContext.Implicits.global
+
+class PeriodicSubmissionControllerSpec extends ControllerUnitTest
+  with MockPropertyService
+  with MockAuthorisedAction
+  with FakeRequestProvider {
+
+  private val underTest = new PeriodicSubmissionController(
+    mockPropertyServices,
+    mockAuthorisedAction,
+    cc
+  )
+
+  ".getPeriodicSubmissionData" should {
+
+
+    val periodicSubmissionResponse = PeriodicSubmissionResponse(List(PeriodicSubmission("submissionId", "2022-11-11", "2023-12-22")))
+
+    "return aPeriodicSubmission when IntegrationFrameworkService returns Right(aPeriodicSubmission)" in {
+      mockAuthorisation()
+      mockGetPeriodicSubmission("2023-24",
+        "taxableEntityId",
+        "incomeSourceId",
+        Right(periodicSubmissionResponse))
+
+      val result = await(underTest.getPeriodicSubmission("2023-24", "taxableEntityId", "incomeSourceId")(fakeGetRequest))
+
+      result.header.status shouldBe OK
+      Json.parse(consumeBody(result)) shouldBe Json.toJson(periodicSubmissionResponse)
+    }
+
+    "return not found when PeriodicSubmissionService returns Left(DataNotFoundError)" in {
+      mockAuthorisation()
+      mockGetPeriodicSubmission("2023-24",
```

Comment:
According to the OpenSpec API, the format for the taxyear is `23-24`. See the example they provide:
<img width="821" alt="image" src="https://github.com/hmrc/income-tax-property/assets/108010791/981dfd99-26a2-48b0-8619-f30cb27951c8">

Let's fix this references

Reply:
Done